### PR TITLE
MAGECLOUD-1581 change base template

### PR DESCRIPTION
**Pull request by hguthrie**
Completed a task to update the Project structure topic to include information on changing the base template. 

-  added steps to change base template
-  changed the section locations and headings to follow a logical flow
-  updated the log file locations
-  updated the example output
-  clarified the section on ignoring files
-  formatted the markdown table purely for readability (no affect on content)

whatsnew
Added section to Cloud [Project structure](https://devdocs.magento.com/guides/v2.2/cloud/project/project-start.html) about how to change the base template.

### CODE REVIEW

File: guides/v2.2/cloud/project/project-start.md [hunk 1]
```diff
@@ -7,47 +7,34 @@ functional_areas:
...
+`.gitignore`              | Specify which files and directories to ignore. See [`.gitignore` reference](#ignoring-files).
+`.magento.app.yaml`       | Configuration file that defines the properties to build your application. See [Configure environments]({{ page.baseurl }}/cloud/env/environments.html).
+`.magento.env.yaml`       | Configuration file that defines actions for the build, deploy, and post-deploy phases. The ece-tools package includes a sample of this file with detailed descriptions for the available variables. See [Configure environments]({{ page.baseurl }}/cloud/env/environments.html).
+`composer.json`           | Fetches the Magento Enterprise Edition and the necessary configuration scripts to prepare your application. See [Prepare your Magento install]({{ page.baseurl }}/cloud/setup/first-time-setup-import-prepare.html).
+`composer.lock`           | Stores version dependencies for every package.
+`magento-vars.php`        | A file used to define [multiple stores]({{ page.baseurl }}/cloud/project/project-multi-sites.html#modify-the-magento-varsphp-file) and sites using [Magento variables]({{ page.baseurl }}/config-guide/multi-site/ms_over.html).
```
Comment (meker12) [inline guides/v2.2/cloud/project/project-start.md]: These link targets seem confusing --- 

multiple stores ==>  *Modify the `magento-vars.php` file* topic in Cloud docs
Magento variables ==> *Multiple websites or stores*  overview topic in Configuration Guide

Please check

File: guides/v2.2/cloud/project/project-start.md [hunk 2]
```diff
@@ -60,61 +47,96 @@ In Integration, Staging, and Production, *only* the following directories are wr
...
+```bash
+git add <path/filename> -f
+```
+
+## Logs
+
+The `/var/log` directory contains logs for all environments. Use the `magento-cloud ssh` command to log in to the environment to access the `/var/log` directory.
 
-Magento logs are located in the `<magento root dir>/var/log` directory.
+For Pro, the deployment log for Staging and Production is in `/var/log/platform` directory. Magento-specific logs are in the `<magento-root-dir>/var/log` directory.
```
Comment (meker12) [inline guides/v2.2/cloud/project/project-start.md]: in `/var/log/platform` directory ==> in the `/var/log/platform` directory

File: guides/v2.2/cloud/project/project-start.md [hunk 3]
```diff
@@ -60,61 +47,96 @@ In Integration, Staging, and Production, *only* the following directories are wr
...
-## Build logs {#build-log}
+### Build logs
 
-After pushing to your environment, you can see the results of the both hooks. Logs from the build hook are redirected to the output stream of `git push`, so you can observe them in the terminal or capture them (along with error messages) with `git push > build.log 2>&1`.
+After pushing changes to your environment, you can observe the build results in the terminal or use SSH to log in and view them in the `var/log/cloud.log` file.
 
-### Deploy logs {#deploy-log}
+### Deploy logs
 
 You can review these logs via SSH into the environment. Change to the directories listed below to review the logs.
```
Comment (meker12) [inline guides/v2.2/cloud/project/project-start.md]: You can review .... via SSH into ...
Better if it was more like the sentence on line 69 above ... use SSH to log in -- also "these logs" ==> deploy logs

File: guides/v2.2/cloud/project/project-start.md [hunk 4]
```diff
@@ -60,61 +47,96 @@ In Integration, Staging, and Production, *only* the following directories are wr
...
+    [2019-01-03 19:44:11] INFO: Backup /app/app/etc/env.php.bak for /app/app/etc/env.php was created.  
+    [2019-01-03 19:44:11] INFO: Backup /app/app/etc/config.php.bak for /app/app/etc/config.php was created.  
+    [2019-01-03 19:44:11] INFO: php ./bin/magento cache:flush --ansi --no-interaction   
+    [2019-01-03 19:44:32] INFO: Warming up failed: http://integration-id-project.us.magentosite.cloud/
+    [2019-01-03 19:44:32] NOTICE: Post-deploy is complete.  
+```
+{: .no-copy}
 
-The value of `<project ID>` depends on the project ID and whether the environment is Staging or Production. For example, with a project ID of `yw1unoukjcawe`, the Staging environment user is `yw1unoukjcawe_stg` and the Production environment user is `yw1unoukjcawe`.
+The deploy log contains start and stop messages for each hook, such as `Start deploy.` and `Deployment complete.`.
```
Comment (meker12) [inline guides/v2.2/cloud/project/project-start.md]: the periods in the message examples seem like punctuation mistakes -- consider leaving the period off in the message for easier reading, or rephrase

File: guides/v2.2/cloud/project/project-start.md [hunk 5]
```diff
@@ -60,61 +47,96 @@ In Integration, Staging, and Production, *only* the following directories are wr
...
+### Application logs
 
-For Production, you have a three node structure. Logs are available with specific information for that node. For example, on the Production environment for project `yw1unoukjcawe`, the deploy log is located at node 1 `/var/log/platform/1.yw1unoukjcawe/post_deploy.log`, node 2 `/var/log/platform/2.yw1unoukjcawe/deploy.log`, and node 3 `/var/log/platform/3.yw1unoukjcawe/post_deploy.log`.
+To review other application logs in Staging or Production, you can access and review those logs in `/var/log/platform/ProjectID`.
 
-Logs for all deployments that have occurred on this environment are appended to this file. Check the timestamps on log entries to verify and locate the logs you want for a specific deployment.
+For Pro plan Staging, the project ID has `_stg` at the end. For example, if you receive `500` errors in Staging and want to review the NGINX logs, you can log in using SSH to the Staging environment and locate the logs in `/var/log/platform/ProjectID_stg`. For Pro plan Production, there are three nodes to check for logs.
```
Comment (meker12) [inline guides/v2.2/cloud/project/project-start.md]: you can use SSH to log in to the Staging environment ....  
locate the logs in the `/var/log/platform/ProjectID_stg` directory 

Last sentence makes me think I'm going to see the directory location of the logs for the the three nodes, but they aren't there. (I know you mentioned them in earlier section -- consider clarifying.)

File: guides/v2.2/cloud/project/project-start.md (hunk 3 above)
Comment (hguthrie) [reply guides/v2.2/cloud/project/project-start.md]: It actually SHOULD be like that. Good catch.

File: guides/v2.2/cloud/project/project-start.md (hunk 1 above)
Comment (hguthrie) [reply guides/v2.2/cloud/project/project-start.md]: This is fine for the table. The Config guide link talks about the Admin UI and defining the Magento variables: `Use the MAGE_RUN_TYPE and MAGE_RUN_CODE variables`
The Cloud link explains the implementation for Cloud and you can see those variables in use.

**Comment by hguthrie** [comment]
runnings tests

**Comment by hguthrie** [comment]
running tests

**Pull request merged**